pkg/deploy/api/v1/conversion.go

```diff
@@ -143,24 +206,6 @@ func convert_v1_DeploymentTriggerImageChangeParams_To_api_DeploymentTriggerImage
 	if err := s.Convert(&in.From, &out.From, 0); err != nil {
 		return err
 	}
-	switch in.From.Kind {
-	case "DockerImage":
-		ref, err := imageapi.ParseDockerImageReference(in.From.Name)
-		if err != nil {
-			return err
-		}
-		out.Tag = ref.Tag
-		ref.Tag, ref.ID = "", ""
-		out.RepositoryName = ref.String()
```

Comment:
I THINK this is the only place that RepositoryName was set for the internal model (and latest referenced in the generator). If we no longer need to support stored triggers of this kind, we can remove the field.